src/components/icons/Telegram.tsx

```diff
@@ -0,0 +1,31 @@
+export default function Telegram({
+  size,
+  style,
+  color,
+}: {
+  size?: number
+  style: React.CSSProperties
+  color: string
+}) {
+  const widthToHeight = 71 / 55
+  const height = size ?? 55
+
+  return (
+    <svg
+      width="17"
+      height="15"
+      style={{ height, width: widthToHeight * height, ...style }}
+      viewBox="0 0 17 15"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        id="telegram-1"
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M13.022 14.0102C13.2501 14.1717 13.544 14.2121 13.8061 14.113C14.0682 14.0131 14.2608 13.7892 14.3189 13.518C14.9344 10.6251 16.4276 3.30307 16.9879 0.671606C17.0304 0.473272 16.9595 0.267147 16.8037 0.134689C16.6479 0.00223004 16.4318 -0.0360201 16.2392 0.0355217C13.2692 1.13486 4.12269 4.56673 0.38419 5.95011C0.146903 6.03794 -0.00750968 6.26603 0.000281807 6.51607C0.00878161 6.76682 0.177361 6.98428 0.420314 7.05724C2.0969 7.55874 4.29764 8.25645 4.29764 8.25645C4.29764 8.25645 5.32612 11.3625 5.86231 12.9421C5.9296 13.1404 6.08473 13.2962 6.28943 13.3501C6.49343 13.4032 6.71159 13.3472 6.86388 13.2035C7.72519 12.3903 9.05683 11.133 9.05683 11.133C9.05683 11.133 11.5869 12.9881 13.022 14.0102ZM5.22341 7.86403L6.41268 11.7868L6.67688 9.30266C6.67688 9.30266 11.2717 5.15819 13.8911 2.7959C13.9676 2.72648 13.9782 2.61032 13.9145 2.52886C13.8514 2.4474 13.7353 2.42827 13.6481 2.48352C10.6123 4.42224 5.22341 7.86403 5.22341 7.86403Z"
+        fill={color}
```

Comment:
Not sure why this svg export is acting differently, but, this was the only way I could get the svg fill to work